config/initializers/1_fast_gettext.rb

```diff
@@ -15,6 +15,11 @@
 # work in all domains context by default (for plugins)
 include FastGettext::TranslationMultidomain
 
+# Keep TRANSLATORS comments
+Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=TRANSLATORS:]
+# Disable fuzzy .po merging
+Rails.application.config.gettext_i18n_rails.msgmerge = %w[--no-fuzzy-matching]
```

Comment:
Just that we don't intend to use fuzzy translation matching.  Can you provide the specifics of what string and translation is in each file as you extract, pull and merge?